examples/custom_messages.py

```diff
@@ -0,0 +1,37 @@
+from locust import HttpUser, task, events, between
```

Comment:
Can we do a more "real world" example? Like the worker asking for some test data (a customer ID or something) and the master replying with it? I think it is one of the most common use cases for this.

Reply:
Sure, let me add in something a little less theoretical